integration/setup/setup_compat.go

```diff
@@ -82,6 +82,10 @@ func SetupCompatWithOpts(tb testing.TB, opts *SetupCompatOpts) *SetupCompatResul
 	// Add the backend's name to prevent the usage of the same database.
 	opts.databaseName = testutil.DatabaseName(tb) + "_" + suffix
 
+	// When database name is too long, database is created but inserting documents
+	// fail with InvalidNamespace error.
+	require.Less(tb, len(opts.databaseName), 64, "database name is too long")
```

Comment:
For other usage of [DatabaseName(...)](https://github.com/FerretDB/FerretDB/blob/main/internal/util/testutil/db.go#L73), the length is checked. In compat test however, `_pg` or  `_tigris` suffix is added after length is checked and the failure is silently ignored.